packages/ckeditor5-ckbox/src/ckboxediting.ts

```diff
@@ -441,3 +445,40 @@ function shouldUpcastAttributeForNode( node: Node ) {
 function isLibraryLoaded(): boolean {
 	return !!window.CKBox;
 }
+
+/**
+ * Checks is access alowed to upload assets, if not it disables `imageUpload` and `ckboxImageEdit` commands.
+ */
+async function verifyUploadAssetsPermission( editor: Editor ) {
+	const ckboxUtils = editor.plugins.get( CKBoxUtils );
+	const origin = editor.config.get( 'ckbox.serviceOrigin' );
+
+	const url = new URL( origin + '/permissions' );
```

Comment:
I'd suggest this approach to building URL:

https://github.com/ckeditor/ckeditor5/blob/ad0dc720566f5ffd38de1fdaa048d87025b6f9a6/packages/ckeditor5-ckbox/src/ckboximageedit/ckboximageeditcommand.ts#L309